.github/workflows/dhis2-verify-lib.yml

```diff
@@ -21,15 +21,15 @@ jobs:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v1
               with:
-                  node-version: 14.x
+                  node-version: 20.x
 
             - uses: c-hive/gha-yarn-cache@v1
             - run: |
                   yarn install --frozen-lockfile
-                  yarn setup
+                  NODE_OPTIONS=--openssl-legacy-provider yarn setup
```

Comment:
wouldn't it be more consistent to add `NODE_OPTIONS=--openssl-legacy-provider` as a an env variable applying to all the steps? and we get rid of it, once webpack/storybook is upgraded

Reply:
See comment above